crv/crvShape.cc

```diff
@@ -751,6 +751,7 @@ static void swapInvalidEdges(Adapt* a)
 
 static void repositionInvalidEdges(Adapt* a)
 {
+  return;
```

Comment:
@avinmoharana 

This file seems to have changed due to debugging as well. Revert it back to how it looks on the develop branch by using
`git checkout develop -- crv/crvShape.cc` [you should be on core's root directory]